### ISSUE TITLE
Show Response Length

### ISSUE THREAD

**Issue by ngkogkos** [hakluke/hakcheckurl#9]
Hi Luke,

Hope you are well. Thanks for working on this.

I think that showing just the status code without the response length along with it, is maybe half the cherry on the pie :). 

Showing the response length next to the status code would allow people to quickly grep out boring/default/wildcard results or chuck them in a file for later processing etc.

Let me know what you think.

**Comment by hakluke** [comment]
Agreed! I might add this at some point, but to be honest, you could just use ffuf ;)

**Comment by ikewzer0** [comment]
would be nice to have Content-Length in this :)

**Comment by M3hank** [comment]
> Agreed! I might add this at some point, but to be honest, you could just use ffuf ;)

That point is now
